### PR TITLE
[doc](release note) add 2.0.0 alpha1 release note

### PR DESCRIPTION
**Pull request by luzhijing**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luzhijing** [comment]
run buildall

File: docs/en/docs/releasenotes/release-2.0.0Alpha.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+
+# Highlight Features
+
+### 1. Semi-Structured Data Storage and Fast Analysis
+
+- Inverted Index: Supports both fulltext search and normal equal, range query.
+  - Supports fulltext search query
+    - Supports Chinese, English and Unicode standard tokenizer.
+    - Supports both STRING and ARRAY<STRING> types.
```
Comment (hf200012) [inline docs/en/docs/releasenotes/release-2.0.0Alpha.md:40]:
```suggestion
    - Supports both `STRING` and `ARRAY<STRING>` types.
```

**Comment by luzhijing** [comment]
run buildall

**Review by xxiao2018 — APPROVED** [review]
LGTM

**Comment by xxiao2018** [comment]
run buildall

**Review by hf200012 — APPROVED** [review]

**Pull request merged**